src/components/Markdown/markdown.scss

```diff
@@ -97,7 +97,7 @@
 
 .page-ul,
 .page-ol {
-  margin: 0 0 6rem $spacing-lg;
+  // margin: 0 0 6rem 0.8rem;
```

Comment:
should this be removed?

Reply:
yeah, it was throwing the menu off the grid based on the spec.